src/pages/request-assign/RequestRegisteredPage.tsx

```diff
@@ -40,6 +40,7 @@ export const RequestRegisteredPage = () => {
     createdDate: data.createdDate ?? '',
     writerName: data.writer?.name ?? '',
     taskId: Number(taskId),
+    isSavedPost: data.isSavedPost ?? false,
```

Comment:
구냥 저스트 ,, 요런것도 있다는 것을 살포시,,

```suggestion
    isSavedPost: !!data.isSavedPost,
```